src/Core/Components/Base/FluentComponentBase.cs

```diff
@@ -29,6 +30,20 @@ public abstract class FluentComponentBase : ComponentBase, IAsyncDisposable, IFl
     /// </summary>
     internal FluentJSModule JSModule => _jsModule ??= new FluentJSModule(JSRuntime);
 
+    /// <summary>
+    /// Gets the class builder, containing the default margin and padding values.
+    /// </summary>
+    protected virtual CssBuilder DefaultClassBuilder => new CssBuilder(Class)
+        .AddClass(Margin.SpacingToStyle().Class)
```

Comment:
Something feels weird about using `Margin.SpacingToStyle().Class`. Don't you think so? See name proposal in `SpacingExtensions.cs`